.github/workflows/presubmit.yaml

```diff
@@ -55,6 +55,6 @@ jobs:
     - name: Install dependencies
       run: apt install -y clang pkg-config meson libfmt-dev libgflags-dev libjsoncpp-dev protobuf-compiler libgtest-dev libcurl4-openssl-dev libprotobuf-c-dev libgoogle-glog-dev libssl-dev libxml2-dev openssl uuid-dev
```

Comment:
These need to be part of the `cuttlefish-base` workflow now as well.

Reply:
Removed the last commit from this PR as it depends on a change submitted to AOSP.